components/pedigree/resources/src/main/resources/pedigree/view/personHoverbox.js

```diff
@@ -125,6 +139,10 @@ define([
             $super();
 
             this.generateMenuBtn();
+            if (this.getNode().getLifeStatus() == "alive" || this.getNode().getLifeStatus() == "deceased") {
+                this.generateAliveWell();
+            }
+            this.getBoxOnHover().attr({'height': PedigreeEditorParameters.attributes.personHoverBoxRadius * 2 + this.getLabelsOffset()});
```

Comment:
this happens even if life status is not alive or dead when no radio buttons are generated

also it should be the other way around, labelsoffsetshould be based on the sizeof extended howerbox, not howerbox on the size of labels which seem random withotu digging into the code